yarn-error.log

```diff
@@ -1,5 +1,6 @@
 Arguments: 
-  C:\Program Files\nodejs\node.exe C:\Program Files (x86)\Yarn\bin\yarn.js sass
```

Comment:
We can ignore this one.

Reply:
Should I add this as a commit on this pull request?

Reply:
I added to the .gitignore those files, but it didn't work I think

Reply:
If I'm not mistaken, you need to run `git rm --cached [[you file]]`.

Reply:
Yup, its a known issue. Here is a solution.

https://stackoverflow.com/a/1139797/1823841

Reply:
Thanks so much, done and committed